BDArmory/Control/BDModuleOrbitalAI.cs

```diff
@@ -318,7 +395,7 @@ private IEnumerator PilotLogic()
                 {
                     // Vessel is on an escape orbit and has passed the periapsis by over 60s, burn retrograde
 
-                    SetStatus("Correcting Orbit (On escape trajectory)");
+                    SetStatus("Correcting Orbit (On escape trajectory)" + (evadingGunfire ? evasionString : ""));
```

Comment:
Maybe just check for `evadingGunfire` in the `SetStatus` function (if it's not evading missiles) and add the extra string instead of adding it for every call to `SetStatus`?

Similarly, can the `fc.RCSVector = evadingGunfire ? ...` lines be simplified? There's only two variations of that line that I can see.